proto/lbm/collection/v1/event.proto

```diff
@@ -318,4 +322,8 @@ message EventRootChanged {
   string contract_id = 1;
   // token id associated with the token.
   string token_id = 2;
+  // id of the previous root before the change.
```

Comment:
I think the `id` seems to be token id. Is it right? If so, please add more detail clearly.